doc/developer/debugging.rst

```diff
@@ -104,8 +106,13 @@ Multiple dev on one server
 
 When you want to run multiple instances on the same server, you should:
 
-- Use a different docker tag for each instance
-- Use a different project name for each instance
+- Use a different docker tag for each instance (``DOCKER_TAG`` in the file ``.env`` files,
+  used on the build and on the run)
+- Use a different project name for each instance (``COMPOSE_PROJECT_NAME`` in the
+  ``.env`` or option ``-p`` of ``docker-compose``)
+- Use a defferent port for each instance (``DOCKER_PORT`` in the ``.env``)
+- It you want to serv the throw the same Apache they shoud have defferant entrypoint
```

Comment:
Not 100% sure what we are saying here. Is it:
- If you want to serve your instance through the same Apache server as the other instances, the instances must have different entry points.

Reply:
Is 'If you want to serve your instances through the same Apache server, each instances must have different entry points. (``VISIBLE_ENTRY_POINT`` in the ``.env``)' better?

Reply:
Yes very good
Except "each instances" -> "each instance"